source/sdk/io/native/FileWin32.ooc

```diff
@@ -101,9 +98,7 @@ version(windows) {
 			// our definition of a file: neither a directory or a link
 			// (and no, FILE_ATTRIBUTE_NORMAL isn't true when we need it..)
 			(ffd, ok) := _getFindData()
-			return (ok) &&
-					(((ffd attr) & FILE_ATTRIBUTE_DIRECTORY) == 0) &&
-					(((ffd attr) & FILE_ATTRIBUTE_REPARSE_POINT) == 0)
```

Comment:
Okay... I think it read a lot more clearly before you put it all on one line, though. What if you keep it the way it was, but fix the indentation?